Flora/Flora/API/APIManager.m

```diff
@@ -29,11 +29,10 @@
 #define HIGH_TEMP_BOUND 100
 
 #ifdef DEBUG
```

Comment:
this is defined in each file...
Instead maybe just define it in APIManager.h   ..
or you could make a Elog.h file (and maybe just have APIManager.h import it...).